lib/client/ipam.go

```diff
@@ -513,7 +513,7 @@ func (c ipams) ReleaseHostAffinities(host *string) error {
 
 // ReleasePoolAffinities releases affinity for all blocks within
 // the specified pool across all hosts.
-func (c ipams) ReleasePoolAffinities(pool common.IPNet) error {
+func (c ipams) ReleasePoolAffinities(pool *common.IPNet) error {
```

Comment:
I think the right thing is to pass a struct, not a pointer.

Don't want the value to accidentally be changed in ReleasePoolAffinities.  I will update.